storage/tianmu/mm/memory_handling_policy.h

```diff
@@ -39,22 +39,23 @@ class MemoryHandling {
   friend class TraceableObject;
 
   using PtrHeapMap = std::unordered_map<void *, HeapPolicy *>;
-  std::unordered_map<TraceableObject *, PtrHeapMap *> m_objs;
+  std::unordered_map<TraceableObject *, PtrHeapMap *> m_objs_;
 
-  HeapPolicy *m_main_heap, *m_huge_heap, *m_system, *m_large_temp;
-  int main_heap_MB, comp_heap_MB;
-  bool m_hard_limit;
-  int m_release_count = 0;
-  size_t m_release_total = 0;
+  HeapPolicy *m_main_heap_, *m_huge_heap_, *m_system_, *m_large_temp_;
+  int main_heap_size_, comp_heap_size_;  // MB
+  bool m_hard_limit_;
```

Comment:
Do init in class definition just like other variables does.

Reply:
ok